csen/download-models.sh

```diff
@@ -10,7 +10,6 @@ do
 		wget -nc http://data.statmt.org/bergamot/models/8bit-students/$lang/$model/model.alphas.npz
 		wget -nc http://data.statmt.org/bergamot/models/8bit-students/$lang/$model/model.intgemm.bin
 		wget -nc http://data.statmt.org/bergamot/models/8bit-students/$lang/$model/model.intgemm.alphas.bin
-		wget -nc http://data.statmt.org/bergamot/models/8bit-students/$lang/$model/lex.s2t.gz
```

Comment:
I think we should leave those to be downloaded, so people can generate their own binary shortlists and we can package the test archives. Only remove them from the deployment archive building.

Reply:
You are right. Revert those download scripts.